src/appleseed.cli/main.cpp

```diff
@@ -507,11 +507,15 @@ namespace
             }
         }
 
+        SearchPaths resource_searchpaths;
```

Comment:
We spelled search paths with a space (almost) everywhere, so for consistency's sake this should be called `resource_search_paths` (fix everywhere). The only place where we didn't is in `test_searchpaths.cpp`, I'll fix that now.